src/cdomains/intDomain.mli

```diff
@@ -196,6 +196,9 @@ sig
   val is_excl_list: t -> bool
   (* Checks if the element is an exclusion set. *)
 
+  val to_incl_list: t -> int_t list option
+  (* Gives a list representation of the included values if possible. *)
```

Comment:
This (and actually some of the comments above) aren't actually documentation comments which should start with `(**`.